components/participant-review.js

```diff
@@ -0,0 +1,171 @@
+import { css, html } from "../html-css-utils.js";
+
+class BaseParticipantReviewElement extends HTMLElement {
+  constructor() {
+    super();
+    this.render();
+  }
+
+  render() {
+    this.attachShadow({ mode: "open" });
+    this.shadowRoot.innerHTML = this.createCss() + this.createHtml();
+  }
+
+  validateAttributes(requiredAttributes) {
+    requiredAttributes.forEach((attr) => {
+      if (!this.hasAttribute(attr)) {
+        throw new Error(`The "${attr}" attribute is required.`);
+      }
+    });
+  }
+}
+
+class ParticipantReviewList extends BaseParticipantReviewElement {
+  createCss() {
+    return css`
+      .participant-review-list {
+        width: 100%;
+        display: grid;
+        grid-template-columns: minmax(306px, 1fr);
+        gap: 10px;
+        place-content: center;
+        place-items: center;
+        margin-top: 20px;
+
+        @media only screen and (min-width: 768px) {
+          grid-template-columns: repeat(2, minmax(306px, 1fr));
+          margin-top: 30px;
+        }
+
+        @media only screen and (min-width: 1024px) {
+          gap: 10px 9px;
+          margin-top: 40px;
+        }
+      }
+    `;
+  }
+
+  createHtml() {
+    return html`
+      <div class="participant-review-list">
+        <slot></slot>
+      </div>
+    `;
+  }
+}
+
+class ParticipantReview extends BaseParticipantReviewElement {
+  constructor() {
+    super();
+    this.validateAttributes(["authorImgSrc", "content", "author"]);
+  }
+
+  createCss() {
+    return css`
+      article {
+        width: 100%;
+        min-height: 300px;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        border-radius: 10px;
+        background-color: var(--bg-300);
+        color: var(--text-900);
+        padding: 39px 20px;
+
+        @media only screen and (min-width: 768px) {
+          min-height: 280px;
+          padding: 37px 20px;
+        }
+
+        @media only screen and (min-width: 1024px) {
+          min-height: 203px;
+          padding: 41px 51px 31px;
+          flex-direction: row;
+          align-items: flex-start;
+        }
+      }
+      article blockquote,
+      figure {
+        margin: 0;
+      }
+      article blockquote {
+        @media only screen and (min-width: 768px) {
+          margin-top: 16px;
+        }
+      }
+      .participant-review-img {
+        width: 104px;
+        height: 104px;
+        flex-shrink: 0;
+        object-fit: cover;
+
+        @media only screen and (min-width: 1024px) {
+          margin-right: 31px;
+        }
+      }
+      .participant-review-img figure > img {
```

Comment:
하위 엘리먼트를 선택하는 방식이 여기는 ">"인 이유가 따로 있을까요?
```suggestion
      .participant-review-img figure > img {
```

Reply:
이전에 html 코드 구조가 바뀌면서 수정했어야했는데 빼먹은 것 같아요! 수정하여 반영 완료하였습니다~